modules/sql-engine/src/test/java/org/apache/ignite/internal/sql/api/IgniteSqlImplTest.java

```diff
@@ -111,9 +107,6 @@ void itsImpossibleToCreateSessionsAfterFacadeIsStopped() throws Exception {
     private static IgniteSqlImpl newSqlFacade() {
         QueryProcessor queryProcessor = mock(QueryProcessor.class);
 
-        when(queryProcessor.createSession(any()))
-                .thenAnswer(ignored -> new SessionId(UUID.randomUUID()));
-
         return new IgniteSqlImpl("test", queryProcessor, mock(IgniteTransactions.class));
```

Comment:
although it works now if we pass `null` instead of mocks, such an approach doesn't feel safe. Neither QueryProcessor nor IgniteTransactions are marked as `@Nullable`. I don't want this test to fail if someone will decide to add additional assertions to ensure null-safety